tsconfig.base.json

```
@@ -14,7 +14,11 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
-    "paths": {}
+    "paths": {
+      "@ion-todo/todo/shared/todo-utils": [
```

Should not include grouping path in lib name
```suggestion
      "@ion-todo/todo-utils": [
```